doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -873,8 +876,10 @@ A `notification` object contains the following subfields:
   in the notification
 + `callback` : URL pointing to the service which will be invoked when a notification is generated. A NGSIv2
   compliant server must support `http` URL schema, other schemas (e.g. schemas for web sockets) could also be supported.
-+ `throttling`: Minimal period of time (in ISO8601 format) which must elapse between two consecutive notifications. It
++ `throttling`: Minimal period of time in seconds which must elapse between two consecutive notifications. It
   is optional.
++ `count` (not editable, only in GET operations): Number of notifications sent due to this subscription.
++ `lastNotification`: Last notification date in ISO6601 format.
```

Comment:
Fixed (the ISO number in) 18aa61203ef827009f1f4d4b64019cb6025110b8

Regarding field name, lastNotification was decided at the end (see skype chat discussion).